src/wallet/wallet.cpp

```diff
@@ -4398,6 +4402,13 @@ bool CWallet::CreateZerocoinToSigmaRemintModel(string &stringError, int version,
 
     LOCK2(cs_main, cs_wallet);
 
+    const Consensus::Params &params = Params().GetConsensus();
+    if (chainActive.Height() < params.nSigmaStartBlock ||
```

Comment:
We already have `sigma::IsSigmaAllowed` to check if sigma already activated.

Reply:
Sure but this check is cleaner because it shows that height lies within specified window (x >= start && x < start+length)